src/general/StageHUDBase.cs

```diff
@@ -1017,9 +1033,11 @@ protected void UpdatePanelSizing(float delta)
 
         var environmentPanelVBoxContainer = environmentPanel.GetNode<VBoxContainer>("VBoxContainer");
         var compoundsPanelVBoxContainer = compoundsPanel.GetNode<VBoxContainer>("VBoxContainer");
+        var agentsPanelVBoxContainer = agentsPanel.GetNode<VBoxContainer>("VBoxContainer");
```

Comment:
I understand the consistency with previously existing code, but this should probably use exported `NodePath`s, as discussed on discord.

Reply:
I think in this case the consistency and the fact that the panels are all setup the same way here with their child content, means that this current approach should be used here.

Reply:
Well, this may still happen to be changed through Godot, so one would then have to open the file and manually edit all the paths... not sure I'm entirely convinced that this approach is theoretically the best (now, there's consistency playing a role as well).

Reply:
I mean the code variables are suffixed with "VBoxContainer" as well and these grab the child from the related panel variable, I don't see these being anyway really modifiable just with Godot editor anyway even if a NodePath was used. So really, unless this whole thing is refactored, I wouldn't make one of these variables act entirely differently from the rest.